### PR TITLE
Show default avatar on network error for comments

### PR DESCRIPTION
**Pull request by rigelk**

### CODE REVIEW

File: client/src/app/videos/+video-watch/comment/video-comment.component.ts [hunk 1]
```diff
@@ -101,6 +103,11 @@ export class VideoCommentComponent implements OnInit, OnChanges {
       )
   }
 
+  switchToDefaultAvatar ($event: ElementEvent) {
```
Comment (Chocobozzz) [inline client/src/app/videos/+video-watch/comment/video-comment.component.ts]: Just cast the event target to `HTMLInputElement`

File: client/src/app/videos/+video-watch/comment/video-comment.component.ts [hunk 2]
```diff
@@ -101,6 +103,11 @@ export class VideoCommentComponent implements OnInit, OnChanges {
...
+    console.log($event.target)
```
Comment (Chocobozzz) [inline client/src/app/videos/+video-watch/comment/video-comment.component.ts]: ;)

Comment (rigelk) [reply client/src/app/videos/+video-watch/comment/video-comment.component.ts]: 😅

File: client/src/app/shared/misc/utils.ts [hunk 3]
```diff
@@ -4,6 +4,10 @@ import { DatePipe } from '@angular/common'
 import { environment } from '../../../environments/environment'
 import { AuthService } from '../../core/auth'
 
+type ElementEvent = Omit<Event, 'target'> & {
```
Comment (Chocobozzz) [inline client/src/app/shared/misc/utils.ts]: I think you can remove this

Comment (rigelk) [reply client/src/app/shared/misc/utils.ts]: No: `timestamp-route-transformer.directive.ts` depends on it.

**Pull request merged**